src/lib/Ska/Parse_CM_File.pm

```diff
@@ -699,6 +699,11 @@ sub MM {
         $manvr_hash{roll}          = $1 if ($final_attitude =~ /ROLL\s*\(deg\):\s+(\S+)/);
         $manvr_hash{dur}           = $1 if ($output_data =~ /Duration\s*\(sec\):\s+(\S+)/);
         $manvr_hash{angle}         = $1 if ($output_data =~ /Maneuver Angle\s*\(deg\):\s+(\S+)/);
+        my @init_quat              = ($1,$2,$3,$4) if ($initial_attitude =~ /Quaternion:\s+(\S+)\s+(\S+)\s+(\S+)\s+(\S+)/);
```

Comment:
Eventually the maneuver summary file will not be part of checking ACA, but for now it's OK.  Alternatively your original idea of reading the ATTITUDE.txt file with astropy.Table will work.  You just need to read it then something like:
```
i0 = np.searchsorted(atts['gretatime'], DateTime(first_backstop_cmd_time).greta)
att0 = [atts[i0 - 1][x] for x in 'q1', 'q2', 'q3', 'q4']
```